src/protos/stub.proto

```diff
@@ -0,0 +1,174 @@
+package bicyclade;
```

Comment:
in proto we can include others prots: import "myproject/base.proto"
we may consider it bt can be done in another PR